File: Swift/프로그래머스/0/181855. 문자열 묶기/문자열 묶기.swift
import Foundation

func solution(_ strArr:[String]) -> Int {
    var numCount = [Int: Int]()
    
    for str in strArr {
        var length = str.count
        
        if let count = numCount[length] {
            numCount[length] = count + 1
        } else {
            numCount[length] = 1
        }
    }
    
    return numCount.values.max() ?? 0
}

File: Python3/프로그래머스/0/181855. 문자열 묶기/문자열 묶기.py
def solution(strArr):
    num_count = {}
    
    for string in strArr:
        length = len(string)
        
        if length not in num_count:
            num_count[length] = 1
        else:
            num_count[length] += 1
            
    
    return max(num_count.values())